Snippets/ServiceControlContracts/ServiceControlContracts_1/MessageFailedHandler.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace Core5.Monitoring
+﻿namespace Core6.Monitoring
```

Comment:
Not that it really impacts anything, but this namespace seems really strange. What does "Monitoring" have to do with this? Also, why is "Core" in here at all?